common/djangoapps/student/constants.py

```diff
@@ -0,0 +1,3 @@
+import pycountry
+
+LANGUAGE_CHOICES = sorted({lang.name for lang in pycountry.languages if hasattr(lang, 'alpha_2')})
```

Comment:
Could you please explain why the condition for `alpha_2` was added in this line or what is meant by `alpha_2`?

Reply:
This checks if the language has a 2 letter language code e.g 'en' for english. If so, only then is it kept as I was maintaining uniformity and that is the usual standard.